src/gluonnlp/models/albert.py

```diff
@@ -38,37 +39,109 @@
 from ..base import get_model_zoo_home_dir, get_repo_model_zoo_url, get_model_zoo_checksum_dir
 from ..utils.config import CfgNode as CN
 from ..utils.misc import load_checksum_stats, download
+from ..utils.registry import Registry
 from ..initializer import TruncNorm
 from ..attention_cell import gen_self_attn_mask
 from ..layers import get_activation, PositionalEmbedding
 from ..op import select_vectors_by_position
 from ..data.tokenizers import SentencepieceTokenizer
 
+albert_cfg_reg = Registry('albert_cfg')
+
+
+@albert_cfg_reg.register()
+def google_albert_base():
+    cfg = CN()
+    # Model Parameters
+    cfg.MODEL = CN()
+    cfg.MODEL.vocab_size = 30000
+    cfg.MODEL.embed_size = 128
+    cfg.MODEL.units = 768
+    cfg.MODEL.hidden_size = 3072
+    cfg.MODEL.max_length = 512
+    cfg.MODEL.num_heads = 12
+    cfg.MODEL.num_layers = 12
+    cfg.MODEL.pos_embed_type = 'learned'
+    cfg.MODEL.activation = 'gelu(tanh)'
+    cfg.MODEL.layer_norm_eps = 1E-12
+    cfg.MODEL.num_groups = 1
+    cfg.MODEL.num_token_types = 2
+    cfg.MODEL.hidden_dropout_prob = 0.0
+    cfg.MODEL.attention_dropout_prob = 0.0
+    cfg.MODEL.dtype = 'float32'
+    cfg.MODEL.layout = 'NT'
+    # Hyper-parameters of the Initializers
+    cfg.INITIALIZER = CN()
+    cfg.INITIALIZER.embed = ['truncnorm', 0, 0.02]
+    cfg.INITIALIZER.weight = ['truncnorm', 0, 0.02]  # TruncNorm(0, 0.02)
+    cfg.INITIALIZER.bias = ['zeros']
+    # Version of the model. This helps ensure backward compatibility.
+    # Also, we can not use string here due to https://github.com/rbgirshick/yacs/issues/26
+    cfg.VERSION = 1
+    cfg.freeze()
+    return cfg
```

Comment:
@ZheyuYe I'm moving the cfgs to the file to make is consistent to Transformer + RoBERTA.

Reply:
ok, I am going to revise other models

Reply:
ok, I am going to revise other models